public/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Gallery",
+  "description": "Share your collection with the world.",
+  "iconPath": "https://gallery.so/favicon.ico"
+}
```

Comment:
added this so that we can be recognized as a "Safe App" by gnosis safe. there's more we can do but this at least gets us over the first step

Reply:
nice! great idea